Scrollbar display fixes

myabc opened the pull request:
https://www.openproject.org/work_packages/15161

myabc committed:
Remove global overflow-y on <html> element

Bug # 15161

Signed-off-by: Alex Coles <alex@alexbcoles.com>

myabc committed:
Prohibit <body>-overflow in absolute layout mode

Signed-off-by: Alex Coles <alex@alexbcoles.com>

ulferts commented:
While I cannot reproduce the failing specs on my machine, this fix is preventing the user from performing some actions from the context menu on work packages at the bottom of the work package table. As the context menu opens downwards, and this in now cut off, the options become unreachable.

![image](https://cloud.githubusercontent.com/assets/617519/4008048/edccf272-29d0-11e4-82a7-afccbdb60f09.png)

myabc committed:
Start xfvb directly on Travis, customise resolution

This should fix specs that only fail on Travis, or when run headless.

See http://docs.travis-ci.com/user/gui-and-headless-browsers/#Configuring-xvfb-screen-size-and-more

/ HT @hschink

Signed-off-by: Alex Coles <alex@alexbcoles.com>

myabc committed:
xvfb on Travis: adjust pidfile path

myabc committed:
xvfb on Travis: unquieten start-stop-daemon

myabc committed:
xvfb on Travis: show additional debug info

myabc committed:
xvfb on Travis: try larger resolution

myabc commented:
I'm really perplexed by the spec failures on Travis. I've tried to reproduce the failures locally with Firefox 31.0, Firefox 24.4 and 24.5 (the version used by Travis).

Firefox 24.x result in other failures (see https://gist.github.com/myabc/9a4148c31ef15fccf4dd), but not this issue.

Additionally, @hschink ran the specs heedlessly locally (using a newer Firefox version).

Commit:
Increase virtual display resolution

myabc commented:
In the end we just needed to increase the resolution – thanks @hschink for helping reproduce the issue!
